src/app/getting-started/services/provider.service.ts

```diff
@@ -63,6 +65,21 @@ export class ProviderService {
     this.redirectToAuth(url);
   }
 
+  /**
+   * 
+   * @param url 
+   */
+
+   getLegacyLinkingUrl(provider: string, redirect: string): string{
```

Comment:
It's not clear why getLegacyLinkingUrl was created? It's almost identical to the link method -- why not just change that instead to use the correct URL params?

Reply:
I didn't want to touch and remove existing code because I'm unsure what the fate of the GettingStarted page would finally be :)